formatter/license-header.txt

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
```

Comment:
are you sure we need anything other than first two lines in the license header? I think we normally use just those two 2 lines.

Reply:
Yeah I think I can remove that. Just took reference from legacy plugin job scheduler . There they use it so I used it as well